Cargo.toml

```diff
@@ -21,10 +21,12 @@ exclude = ["examples/rust_blank/ext/rust_blank", "examples/custom_exception_ruby
 embed = ["rb-sys/link-ruby"]
 rb-sys-interop = []
 ruby-static = ["rb-sys/ruby-static"]
+bytes-crate = ["bytes"]
 
 [dependencies]
 magnus-macros = { version = "0.4.0", path = "magnus-macros" }
 rb-sys = { version = "0.9.56", default-features = false, features = ["bindgen-rbimpls", "bindgen-deprecated-types"] }
+bytes = { version = "1.4.0", optional = true }
```

Comment:
Could you use `version = "1"` for the version here? That way if someone is using a crate with Magnus that has a dependency on an older (but still 1.x.x) version of `bytes` then cargo can just use that single version across the two crates.